test/test_mapper.cc

```diff
@@ -24,100 +26,46 @@ using namespace Legion::Mapping;
 /// Mapper
 ///
 
-static LegionRuntime::Logger::Category log_fft_test_mapper("fft_test_mapper");
+static Logger log_fft_test_mapper("fft_test_mapper");
 
-class FFTTestMapper : public DefaultMapper
-{
+// extend Default mapper
```

Comment:
Don't transliterate the code.

Reply:
Removed these comments - they were also absent in the original repo.